### ISSUE TITLE
Video is stretch while playing video then changing to another video

### ISSUE THREAD

**Issue by congtuyenvip** [BrightcoveOS/android-player-samples#161]
@sergiomartinez4 

My situation is:
My brightcove video view is match parent (which device screen size). When video is playing, then change to other video source with difference source size, so video screen is stretched to match parent size. After video is loaded, then surface view size match video size.

My question is:
How can I make surface view not stretch like this, only reize when video is loaded?
I see there is event video size know and unknown. 

Thanks

**Comment by sergiomartinez4** [comment]
@congtuyenvip 
Yes, there is precisely an event named `EventType.VIDEO_SIZE_KNOWN` that you can listen and react to.
Please take a look at these forum posts where we talk about a similar issue as the one you're explaining:
https://groups.google.com/d/msg/brightcove-native-player-sdks/4fbEEszqtrk/cpVGQmypAQAJ
https://groups.google.com/d/msg/brightcove-native-player-sdks/4mcA6fhf1fU/stIrhg1RAAAJ
https://groups.google.com/d/msg/brightcove-native-player-sdks/D04MISD2Ous/xeA59CIUAgAJ

Hope that helps!
